### DISCUSSION TITLE
EV: Schnellladen per GPI aktivieren?

### DISCUSSION THREAD

**Discussion by solanjo** [camueller/SmartApplianceEnabler#235] [Allgemein]
Hallo Axel,

leider lebe ich in einem Haushalt mit Menschen zusammen, die nicht so technik-affin sind wie ich... ;-)

Wir haben an unserer Wallbox einen Taster, mit dem ich den EV-CC auf enable setzen kann. So lässt sich falls es mal schnell gehen soll, das Laden ad-hoc starten, der SAE wir quasi übersteuert.
Bedauerlicherweise kann es allerdings vorkommen, dass der zuletzt gesetzte Ladestrom sehr gering ist und somit eine vermeintliche "Schnellladung" ziemlich langsam wird. Es ist mir auch schon aufgefallen, dass der SEA weiterhin ins Stromstärkeregister schreibt und somit der Ladestrom nicht auf maximal bleibt.

Gäbe es technisch die Möglichkeit, den SAE per GPI-Taster zu steuern? In meinem Fall also ein Setzen der Ampel auf "Schnellladen" ausgelöst wird? Vielleicht ließe sich sowas auch per Skript steuern, das durch ein GPI aktiviert wird.
Sicherlich ein nice-to-have. Für meine Frau allerdings die Bewahrung vor weiteren Schrecken...

**Comment by Trunks1982** [comment]
Das finde ich ne gute Idee.Vielleicht nicht nur über GPIO.Per Script wäre super das man auch zigbee taster über iobroker integrieren könnte.👍

**Reply by Trunks1982** [reply]
> Das finde ich ne gute Idee.Vielleicht nicht nur über GPIO.Per Script/HTTP Befehl wäre super das man auch zigbee taster über iobroker integrieren könnte.👍

**Comment by camueller** [comment]
Der SAE an via REST-Calls gesteuert werden - nichts anderes sind ja die Klicks auf die Ampel. Diese REST-Calls kann man ja auch anders auslösen, z.B. via Taster. Allerdings sehe ich das eher nicht als SAE-Feature, sondern als etwas, das man sich per Node RED oder anderen Automatisierungstools basteln kann.

**Reply by solanjo** [reply]
Danke für den Hinweis!
Du hast das ja auch ein wenig dokumentiert. Würde das dann für den EV-Charger bedeuten, dass nur noch ein Attribut <RecommendedPowerConsumption> eingefügt werden muss? Und ein Timeframe vorher noch definieren?

**Reply by camueller** [reply]
Du kannst ja einfach im Browser mit den Entwickler-Tools schauen, welche Rest-Calls das Web-Frontend macht, wenn Du auf die grün Ampel und "Starten" klickst.
